Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -409,27 +413,27 @@ class EndAsyncGeneratorFunctionDefinition: EndAnyFunctionDefinition {}
 
 class Return: Operation {
     init() {
-        super.init(numInputs: 1, numOutputs: 0, attributes: [.isJump])
+        super.init(numInputs: 1, numOutputs: 0, attributes: [.isJump], contextRequired: ProgramContext([.function, .generatorFunction, .asyncFunction]))
     }
 }
 
 // A yield expression in JavaScript
 class Yield: Operation {
     init() {
-        super.init(numInputs: 1, numOutputs: 1, attributes: [])
+        super.init(numInputs: 1, numOutputs: 1, attributes: [], contextRequired: ProgramContext.generatorFunction)
```

Comment:
I think here and elsewhere in this file you can just write `requiredContext: .generatorFunction` instead of `requiredContext: ProgramContext.generatorFunction`